docs/source/teams/cloud_media.rst

```diff
@@ -652,6 +652,42 @@ collection being annotated, then you can simply pass `cloud_manifest=True`:
     The cloud manifest file must contain all media files in the sample
     collection being annotated.
 
+.. _teams-annotating-cloud-media-labelbox:
+
+Annotating cloud-backed datasets with Labelbox
+______________________________________________
+
+When using FiftyOne to
+`annotate data with Labelbox <https://voxel51.com/docs/fiftyone/integrations/labelbox.html>`_,
```

Comment:
nit: can you use this link style instead?

```
:ref:`annotate data with Labelbox <labelbox-integration>`
```

Reply:
Good call, also updated the CVAT ref :+1: